src/wallet/rpcwallet.cpp

```diff
@@ -4877,7 +4879,7 @@ UniValue z_sendmany(const UniValue& params, bool fHelp)
             "      \"amount\":amount    (numeric, required) The numeric amount in " + CURRENCY_UNIT + " is the value\n"
             "      \"memo\":memo        (string, optional) If the address is a zaddr, raw data represented in hexadecimal string format\n"
             "    }, ... ]\n"
-            "3. minconf               (numeric, optional, default=1) Only use funds confirmed at least this many times.\n"
+            "3. minconf               (numeric, optional, default=%u) Only use funds confirmed at least this many times.\n"
```

Comment:
It'd be more clear if this did `" + strprintf("%s", FormatMoney(nAnchorConfirmations)) + "` similar to the line below it.